package.json

```diff
@@ -12,6 +12,7 @@
     "lint--fix": "eslint . --fix --report-unused-disable-directives",
     "merge": "ts-node tools/src/merger/merge.ts",
     "test": "npm run test:unit && npm run test:integ",
+    "jest": "jest",
```

Comment:
I put this back because to run an individual test you can't do `npm run test:unit -- ...test.ts`, it has `testMatch` flags, but you can do `npm run jest -- ...test.ts`.